recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ test:
   imports:
     - pandasgui  # [win]
   commands:
-    - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python -c "import pandasgui"'  # [not win]
+    - DISPLAY=localhost: 1.0 xvfb-run -a bash -c 'python -c "import pandasgui"'  # [not win]
```

Comment:
@conda-forge/core Does anyone have an idea why the @regro-cf-autotick-bot always automatically inserts this whitespace here? This always breaks the build process..

Reply:
it uses a yaml parser and this line looks like it could be a yaml dict.

the solution is to add quotes around it to make it a string

you'll need to escape quotes inside when doing this.

Reply:
@beckermr Ok, thanks for the quick reply! That´s a bit strange because there seem to be quite some other feedstocks that use the same syntax: https://github.com/search?q=filename%3Ameta.yaml+DISPLAY%3Dlocalhost%3A1.0

How would the line look like then or where do we have to add the quotes?

Reply:
Try `"DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python -c \"import pandasgui\"'"`

Reply:
or maybe

`DISPLAY="localhost:1.0" xvfb-run -a bash -c 'python -c "import pandasgui"'`

Reply:
Could we simply use `DISPLAY=localhost:1.0 xvfb-run python -c "import pandasgui"  # [not win]`?